test/api/support/ffactory.js

```diff
@@ -0,0 +1,98 @@
+const Promise = require("bluebird")
```

Comment:
Since we've moved to node v6 (yea!), can we start moving to using native [es2015 promises](http://node.green/) instead of bluebird? Mocha's happy with whatever promise implementation we throw at it...